packages/react/src/components/ListItem/README.md

```diff
@@ -0,0 +1,35 @@
+## The component
+
+Single item with texts & right icon, which can be both used to display information or trigger navigation components & actions.
+
+## Usage
+
+```jsx
+import { ListItem, ICONS } from "@heetch/flamingo-react";
+
+export default () => (
+  <>
+    <ListItem
+      title="List Item"
```

Comment:
This has been moved to a `children` prop 🙂

Reply:
Sweet catch

👍